src/App.js

```diff
@@ -1,16 +1,62 @@
 import React from 'react';
+import { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+
+  const [chatLogData, setChatLogData] = useState(chatMessages);
+  const [likes, setLikes] = useState(0);
```

Comment:
This works if all the messages start as unliked, but what would happen if some of the original messages were already liked? Can you think of a way to change your code to use one fewer state variable and avoid this issue?